ide/css.lib/nbproject/org-netbeans-modules-css-lib.sig

```diff
@@ -1101,6 +1102,13 @@ meth protected java.util.List<java.lang.String> postfixes()
 supr org.netbeans.modules.css.lib.api.properties.TokenAcceptor$NumberPostfixAcceptor
 hfds POSTFIXES
 
+CLSS public static org.netbeans.modules.css.lib.api.properties.TokenAcceptor$Flex
```

Comment:
Interesting. CCing @sdedic

Reply:
@matthiasblaesing does the Flex acceptor need to be friend-public  ?

Reply:
I don't know the whole history of the CSS Parser, but for two reasons I'm inclined to keep this as is:
a) the export is friend-only and all users are in-tree
b) there is nothing special in `TokenAcceptor$Flex` that separates it from the other `TokenAcceptors` that are already exported. I see no benefit special casing this new instance.

Reply:
ad (a): friend API only means that there's a chance to trace and update dependents when an API change is made, not a permission to extend API clutter (= add unnecessary stuff).

ad (b) there are at least 3 categories of TokenAcceptors; `TokenImageAcceptor`, `NumberPostfixAcceptor`, which serve as a base and +- could deserve to be `public` for possible extensions. Then there are `Date`, `Integer`, `Number` ... which might represent data types with possibly different handling in clients. And last, there are 'unit' acceptors, which do not interest anyone as they all represent a number. Your `Flex` additionl falls into the last category.

This API is awkward from its inception, because:
- `ACCEPTORS`, `ACCEPTORS_MAP` can be modified externally, making each other and `INSTANCES` inconsistent. No such mutation is in place, but the API is open.
- no of the other 'terminal' unit classes are explicitly used anywhere else: there's no reason they should have ever been public/exported

That said, I do not see 'keeping consistency' as a value in this case. This new addition to the clutter was caught 'in time', before it is released, so the clutter increase could be simply avoided. Please make the class non public. 

For the rest, I've filed [NETBEANS-4977](https://issues.apache.org/jira/browse/NETBEANS-4977) - assigned to myself, so it does not take anyone's precious free time. I'd process the issue when more incompatible changes in web-friends accumulate.

Reply:
I will not create a new PR right now. From my POV the only result is more potential for destabilisation and we are 5 days away. We are 5 days before release, I don't see benefit in rushing this in, given that the new issue will touch the whole structure anyway.